examples/get-started/react/basic/app.js

```diff
@@ -36,31 +32,99 @@ function Root() {
         getLineColor={[60, 60, 60]}
         getFillColor={[200, 200, 200]}
       />
-      <GeoJsonLayer
-        id="airports"
-        data={AIR_PORTS}
-        filled={true}
-        pointRadiusMinPixels={2}
-        pointRadiusScale={2000}
-        getRadius={f => 11 - f.properties.scalerank}
-        getFillColor={[200, 0, 80, 180]}
-        pickable={true}
-        autoHighlight={true}
-        onClick={onClick}
-      />
-      <ArcLayer
-        id="arcs"
-        data={AIR_PORTS}
-        dataTransform={d => d.features.filter(f => f.properties.scalerank < 4)}
-        getSourcePosition={f => [-0.4531566, 51.4709959]}
-        getTargetPosition={f => f.geometry.coordinates}
-        getSourceColor={[0, 128, 200]}
-        getTargetColor={[200, 0, 80]}
-        getWidth={1}
+      <TileLayer
+        // tileSize={128}
+        getTileData={async tile => {
+          const {
+            x,
+            y,
+            bbox: {north, east, south, west},
+            signal
+          } = tile;
+          const wait = 2000;
+          // Math.round((Math.random() * 8) + 2);
+          // console.log("waiting", wait); // eslint-disable-line no-console
+          // return fetch(`https://cors-anywhere.herokuapp.com/https://postman-echo.com/delay/${wait}`);
+          // return new Promise(resolve => setInterval(resolve, wait));
+
+          // docker run --rm -it -p 7000:80 ealen/echo-server
+          await fetch(
```

Comment:
can ignore changes to this example, it was just an easy way to test my changes for now